src/PIL/Image.py

```diff
@@ -2225,8 +2362,9 @@ def toqpixmap(self):
 
 class ImagePointHandler(object):
     # used as a mixin by point transforms (for use with im.point)
-    pass
-
+    def point(self, s):
+        # type (T) -> T
```

Comment:
What is `T` here? This doesn't look familiar from when I was working on this. My instinct is a `TypeVar`, but `T` is not very descriptive?